tests/compute/test_new_kvstore.py

```diff
@@ -0,0 +1,161 @@
+
+from numpy.testing import assert_array_equal
+
+# Create an one-part Graph
+node_map = F.tensor([0,0,0,0,0,0], F.int64)
+edge_map = F.tensor([0,0,0,0,0,0,0], F.int64)
+global_nid = F.tensor([0,1,2,3,4,5], F.int64)
+global_eid = F.tensor([0,1,2,3,4,5,6], F.int64)
+
+gpb = dgl.distributed.GraphPartitionBook(part_id=0,
+                                         num_parts=1,
+                                         node_map=node_map,
+                                         edge_map=edge_map,
+                                         global_nid=global_nid,
+                                         global_eid=global_eid)
+
+node_policy = dgl.distributed.PartitionPolicy(policy_str='node',
+                                              part_id=0,
+                                              partition_book=gpb)
+
+edge_policy = dgl.distributed.PartitionPolicy(policy_str='edge',
+                                              part_id=0,
+                                              partition_book=gpb)
+
+data_0 = F.tensor([[1.,1.],[1.,1.],[1.,1.],[1.,1.],[1.,1.],[1.,1.]], F.float32)
+data_1 = F.tensor([[2.,2.],[2.,2.],[2.,2.],[2.,2.],[2.,2.],[2.,2.],[2.,2.]], F.float32)
+data_2 = F.tensor([[0.,0.],[0.,0.],[0.,0.],[0.,0.],[0.,0.],[0.,0.]], F.float32)
+
+def init_zero_func(shape, dtype):
+    return F.zeros(shape, dtype)
+
+def udf_push(target, name, id_tensor, data_tensor):
+    target[name] = F.scatter_row(target[name], id_tensor, data_tensor*data_tensor)    
+
+def test_partition_policy():
+    assert node_policy.policy_str == 'node'
+    assert edge_policy.policy_str == 'edge'
+    assert node_policy.part_id == 0
+    assert edge_policy.part_id == 0
+    local_nid = node_policy.to_local(F.tensor([0,1,2,3,4,5]))
+    local_eid = edge_policy.to_local(F.tensor([0,1,2,3,4,5,6]))
+    assert_array_equal(F.asnumpy(local_nid), F.asnumpy(F.tensor([0,1,2,3,4,5], F.int64)))
+    assert_array_equal(F.asnumpy(local_eid), F.asnumpy(F.tensor([0,1,2,3,4,5,6], F.int64)))
+    nid_partid = node_policy.to_partid(F.tensor([0,1,2,3,4,5], F.int64))
+    eid_partid = edge_policy.to_partid(F.tensor([0,1,2,3,4,5,6], F.int64))
+    assert_array_equal(F.asnumpy(nid_partid), F.asnumpy(F.tensor([0,0,0,0,0,0], F.int64)))
+    assert_array_equal(F.asnumpy(eid_partid), F.asnumpy(F.tensor([0,0,0,0,0,0,0], F.int64)))
+    assert node_policy.get_data_size() == len(node_map)
+    assert edge_policy.get_data_size() == len(edge_map)
+
+def start_server():
+	# Init kvserver
+    kvserver = dgl.distributed.KVServer(server_id=0,
+                                        ip_config='ip_config.txt',
+                                        num_clients=1)
+    kvserver.init_data(name='data_0', data_tensor=data_0)
+    kvserver.set_partition_policy('data_0', 'node', gpb)
+    kvserver.init_data(name='data_1', data_tensor=data_1)
+    kvserver.set_partition_policy('data_1', 'edge', gpb)
+    # start server
+    server_state = dgl.distributed.ServerState(kv_store=kvserver)
```

Comment:
how does get_server_state return thsi server state?

Reply:
Actually the get_server_state has not been implemented.